03week/ticTacToe.js

```diff
@@ -23,26 +44,106 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
+function onTheBoard(row, column) {
+  if((row == 0 || row == 1 || row == 2) && (column == 0 || column == 1 || column == 2)){
+    isItBlank(row, column);
+  }
+  else{
+    console.log("That location is not on this board!")
+  }
+}
+
+function isItBlank(row, column){
+  if(board[row][column] === " "){
+    makeYourMark(row, column);
```

Comment:
this isn't a pure function, you should de-couple it from on the makeYourMark function. This function should ONLY return a boolean if it's blank, it shouldn't call another function